jdl/converters/jdl-to-json/jdl-without-application-to-json-converter.ts

```diff
@@ -71,21 +72,21 @@ function resetState() {
 }
 
 function setBasicEntityInformation() {
-  const convertedEntities = BasicEntityConverter.convert(jdlObject.getEntities());
+  const convertedEntities = BasicEntityConverter.convert(jdlObject!.getEntities());
```

Comment:
You're right to use the `!` and not fix it in this PR.
We have to create an issue to change how the design of this kind of files (like using a class, or another solution).

Reply:
Yes, I agree that this has to be solved. The need for this non-null assertion comes from the usage of global variables and how they are initialized. Some parts of the code are written in a way that makes it hard to rely on TypeScripts type safety.